src/rust/src/utils/mod.rs

```diff
@@ -501,28 +528,79 @@ pub fn robj_to_usize(robj: extendr_api::Robj) -> RResult<usize> {
     robj_to_u64(robj).and_then(try_u64_into_usize)
 }
 
+fn err_no_nan<T>() -> RResult<T> {
+    rerr().plain("any NA value is not allowed here".to_string())
```

Comment:
`.to_string()` could be omitted since `Into<String>` is implemented for `&str`.

Reply:
done

Reply:
Actually there are multiple occurrences of `.to_string()` that you may want to remove